src/components/application_manager/src/state_controller_impl.cc

```diff
@@ -245,129 +245,185 @@ void StateControllerImpl::SetRegularState(ApplicationSharedPtr app,
 }
 
 namespace {
-bool HasStreamingPermissions(mobile_apis::HMILevel::eType val) {
+bool IsStreamableHMIType(mobile_apis::HMILevel::eType val) {
   using namespace helpers;
   return Compare<mobile_apis::HMILevel::eType, EQ, ONE>(
       val, mobile_apis::HMILevel::HMI_FULL, mobile_apis::HMILevel::HMI_LIMITED);
 }
+
+bool IsSameAudioAppType(const Application& app1, const Application& app2) {
+  // compatibility constants
```

Comment:
@EKuliiev All comments should start from capital letter

Reply:
@EKuliiev I would add some doxygen comment.